demos/object_detection_demo/python/object_detection_demo.py

```diff
@@ -204,6 +204,9 @@ def draw_detections(frame, detections, palette, labels, threshold):
             if isinstance(detection, models.DetectionWithLandmarks):
                 for landmark in detection.landmarks:
                     cv2.circle(frame, (int(landmark[0]), int(landmark[1])), 2, (0, 255, 255), 2)
+    y_scale, x_scale = [i / j for i, j in zip(size, (720, 1280))]
+    scale = max(x_scale, y_scale, 1)
+    frame = cv2.resize(frame, (int(size[1]/scale), int(size[0]/scale)))
```

Comment:
1. `cv2.resize()` reallocates an array even if `scale` is 1: `frame.__array_interface__['data'][0] == resized.__array_interface__['data'][0]` gives `False`. You should handle case when no resize is needed separately.
2. `resize()` is fine computing `dst` size on its own given `fx` and `fy`. There is no need for you to compute `(int(size[1]/scale), int(size[0]/scale))`. Plus I guess it shoul have been `round()`, not `int()`, though I'm not sure :)
3. Don't give a name to `resize()`'s result. Just return what it gives.